src/SSHDebugPS/Docker/DockerHelper.cs

```diff
@@ -53,63 +55,142 @@ public static IEnumerable<DockerContainerInstance> GetLocalDockerContainers(stri
                 {
                     if (!string.IsNullOrWhiteSpace(args))
                     {
-                        if (args.Trim()[0] != '{')
-                        {
-                            // output isn't json, command Error
-                            errorSB.Append(args);
-                        }
-                        else
-                        {
-                            if (DockerContainerInstance.TryCreate(args, out DockerContainerInstance containerInstance))
-                            {
-                                containers.Add(containerInstance);
-                            }
-                            containerCount++;
-                        }
+                        args = args.Trim(charsToTrim);
+                        callback(args);
                     }
                 });
 
                 commandRunner.Start();
 
                 bool cancellationRequested = false;
-                VS.VSOperationWaiter.Wait(UIResources.QueryingForContainersMessage, false, (cancellationToken) =>
+                VS.VSOperationWaiter.Wait(UIResources.RunningDockerCommandMessage, false, (cancellationToken) =>
```

Comment:
Can you pass in the correct resource to use? A generic `Running docker command...` gives no information to the user what is actually happening. 

Add an argument to this method so we can still say `Querying for Containers`. Then also have something like `Getting Docker Container Information...` and `Getting Docker Engine Version Information` for the other two commands.